components/rating/index.css

```diff
@@ -74,37 +121,143 @@ governing permissions and limitations under the License.
     }
   }
 
   /* All stars following the hovered star */
   &:hover ~ .spectrum-Rating-icon {
-    @extend %spectrum-Rating-strokedStar;
+    .spectrum-Rating-starActive {
+      display: none;
+    }
+    .spectrum-Rating-starInactive {
+      display: block;
+    }
   }
 }
 
 .spectrum-Rating-starActive,
 .spectrum-Rating-starInactive {
-  inline-size: var(--spectrum-icon-star-width);
-  block-size: var(--spectrum-icon-star-height);
   margin: 0 auto;
 }
+
+.spectrum-Rating-icon {
+  color: var(--highcontrast-rating-icon-color-default, var(--mod-rating-icon-color-default, var(--spectrum-rating-icon-color-default)));
+
+  &.is-selected {
+    color: var(--highcontrast-rating-icon-color-default, var(--mod-rating-icon-color-default, var(--spectrum-rating-icon-color-default)));
+  }
+
+  &:hover {
+    /* All stars following the hovered star */
+    & ~ .spectrum-Rating-icon {
+      color: var(--highcontrast-rating-icon-color-default, var(--mod-rating-icon-color-default, var(--spectrum-rating-icon-color-default)));
+    }
+  }
+}
+
+.spectrum-Rating--emphasized {
+  &.is-focused {
+    .spectrum-Rating-icon.is-selected {
+      color: var(--highcontrast-rating-emphasized-icon-color-key-focus, var(--mod-rating-emphasized-icon-color-key-focus, var(--spectrum-rating-emphasized-icon-color-key-focus)));
+    }
+  }
+
+  .spectrum-Rating-icon {
+    color: var(--highcontrast-rating-icon-color-default, var(--mod-rating-icon-color-default, var(--spectrum-rating-icon-color-default)));
+    
+    &.is-selected {
+      color: var(--highcontrast-rating-emphasized-icon-color-default, var(--mod-rating-emphasized-icon-color-default, var(--spectrum-rating-emphasized-icon-color-default)));
+    }
+
+    &:hover {
+      /* All stars following the hovered star */
+      & ~ .spectrum-Rating-icon {
+        color: var(--highcontrast-rating-icon-color-default, var(--mod-rating-icon-color-default, var(--spectrum-rating-icon-color-default)));
+      }
+    }
+  }
+
+  &:hover {
+    .spectrum-Rating-icon {
+      /* Make all stars selected when the component is hovered */
+      color: var(--highcontrast-rating-emphasized-icon-color-default, var(--mod-rating-emphasized-icon-color-default, var(--spectrum-rating-emphasized-icon-color-default)));
+
+      &:hover {
+        /* The icon that's actually hovered is darker */
+        color: var(--highcontrast-rating-emphasized-icon-color-hover, var(--mod-rating-emphasized-icon-color-hover, var(--spectrum-rating-emphasized-icon-color-hover)));
+        & ~ .spectrum-Rating-icon {
+          color: var(--highcontrast-rating-icon-color-default, var(--mod-rating-icon-color-default, var(--spectrum-rating-icon-color-default)));
+        }
+      }
+
+      &:active {
+        color: var(--highcontrast-rating-emphasized-icon-color-down, var(--mod-rating-emphasized-icon-color-down, var(--spectrum-rating-emphasized-icon-color-down)));
+      }
+    }
+  }
+}
+
+/* Disabled */
+.spectrum-Rating {
```

Comment:
Should this get tucked in with the other states on the parent class around line 39ish?